Reference libcperciva's headers relatively since Xcode's user header search path setting doesn't seem to work.

--- scrypt~/lib/crypto/crypto_scrypt-ref.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/lib/crypto/crypto_scrypt-ref.c	2015-10-22 11:27:35.000000000 -0400
@@ -33,8 +33,8 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "sha256.h"
-#include "sysendian.h"
+#include "../../libcperciva/alg/sha256.h"
+#include "../../libcperciva/util/sysendian.h"
 
 #include "crypto_scrypt.h"
 
--- scrypt~/lib/crypto/crypto_scrypt.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/lib/crypto/crypto_scrypt.c	2015-10-22 11:27:53.000000000 -0400
@@ -37,7 +37,7 @@
 #include <string.h>
 
 #include "cpusupport.h"
-#include "sha256.h"
+#include "../../libcperciva/alg/sha256.h"
 #include "warnp.h"
 
 #include "crypto_scrypt_smix.h"
--- scrypt~/lib/crypto/crypto_scrypt_smix.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/lib/crypto/crypto_scrypt_smix.c	2015-10-22 11:27:45.000000000 -0400
@@ -29,8 +29,8 @@
 #include <stdint.h>
 #include <string.h>
 
-#include "sha256.h"
-#include "sysendian.h"
+#include "../../libcperciva/alg/sha256.h"
+#include "../../libcperciva/util/sysendian.h"
 
 #include "crypto_scrypt_smix.h"
 
--- scrypt~/lib/scryptenc/scryptenc.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/lib/scryptenc/scryptenc.c	2015-10-22 11:28:17.000000000 -0400
@@ -36,11 +36,11 @@
 #include <unistd.h>
 
 #include "crypto_aes.h"
-#include "crypto_aesctr.h"
+#include "../../libcperciva/crypto/crypto_aesctr.h"
 #include "crypto_entropy.h"
 #include "insecure_memzero.h"
-#include "sha256.h"
-#include "sysendian.h"
+#include "../../libcperciva/alg/sha256.h"
+#include "../../libcperciva/util/sysendian.h"
 
 #include "crypto_scrypt.h"
 #include "memlimit.h"
--- scrypt~/libcperciva/alg/sha256.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/libcperciva/alg/sha256.c	2015-10-22 11:28:07.000000000 -0400
@@ -3,7 +3,7 @@
 #include <string.h>
 
 #include "insecure_memzero.h"
-#include "sysendian.h"
+#include "../util/sysendian.h"
 
 #include "sha256.h"
 
--- scrypt~/libcperciva/crypto/crypto_aesctr.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/libcperciva/crypto/crypto_aesctr.c	2015-10-22 11:27:19.000000000 -0400
@@ -2,7 +2,7 @@
 #include <stdlib.h>
 
 #include "crypto_aes.h"
-#include "sysendian.h"
+#include "../util/sysendian.h"
 
 #include "crypto_aesctr.h"
 
--- scrypt~/libcperciva/crypto/crypto_entropy.c	2015-08-08 14:45:05.000000000 -0400
+++ scrypt/libcperciva/crypto/crypto_entropy.c	2015-10-22 11:26:21.000000000 -0400
@@ -5,7 +5,7 @@
 #include "entropy.h"
 #include "insecure_memzero.h"
 
-#include "sha256.h"
+#include "../alg/sha256.h"
 
 #include "crypto_entropy.h"
 
Only in scrypt/: patched_for_xcode
